src/web100-pcap.c

```diff
@@ -899,7 +899,7 @@ void init_pkttrace(I2Addr srcAddr, struct sockaddr_storage sock_addr[], int sock
 
   /*  device = pcap_lookupdev(errbuf); */
   if (device == NULL) {
-    fprintf(stderr, "pcap_lookupdev failed: %s\n", errbuf);
+    fprintf(stderr, "pcap_lookupdev failed\n", errbuf);
```

Comment:
Looks like a typo: do we need the `errbuf` parameter if the format string is static?